docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,70 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) [Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it:
+
+```console
+$ git clone https://github.com/johndoe/reana.git
+$ cd reana
+$ mkvirtualenv reana && pip install . --upgrade
+```
+
+* (2) Fork all REANA cluster components repositories:
+
+```console
+(reana) $ reana-dev git-fork
```

Comment:
We need `-c ALL` to clone everything.

Also, perhaps change to using eval already? `eval "$(reana-dev git-fork -c ALL)"`

Reply:
I didn't add it because (1) not everyone has Firefox installed and (2) most of the people won't execute an eval of something they don't know, so probably either because (1) of because or (2) they will end up executing just `reana-dev git-fork -c ALL`.

Reply:
WRT (1), people can actually use any browser, e.g. `reana-dev git-fork -c ALL -b chromium` to use Chromium. 

WRT (2), OK, that's good, but then we should add a comment like `# read and run the printed eval` or something like that, making it clear that some "further action" is required of them.